src/main/kotlin/study/racinggame/domain/Race.kt

```diff
@@ -0,0 +1,28 @@
+package study.racinggame.domain
+
+import study.racinggame.service.RaceRule
+
+class Race(
+    private val carStorage: List<Car>,
+    private val raceRule: RaceRule
+) {
+
+    fun runTrack(): List<CarResponse> {
+        moveCarsIfPossible()
+        return carStorage.map(CarResponse.Companion::of)
+    }
+
+    fun getRaceWinner(): String {
+        val maxValue: Int = carStorage.maxOf { each -> each.position() }
+        return carStorage.filter { each -> each.position() == maxValue }
+            .joinToString(", ") { winner -> winner.name }
```

Comment:
```suggestion
        val maxValue: Int = carStorage.maxOf { it.position() }
        return carStorage.filter { it.position() == maxValue }
            .joinToString(", ") { it.name }
```

it을 적극적으로 써주시는 것도 나쁘지 않다고 봅니다!

Reply:
it을 사용하는 것은 좋은 방법이네요!
될수있으면 it  혹은 람다 reference 패턴으로 바꿔보았습니다.

`        val maxValue: Int = carStorage.maxOf(Car::position)
        return carStorage.filter { it.position == maxValue }
            .joinToString(", ", transform = Car::name)`